updated gemfiles rails to 5.0.2

Tonyynot14 opened the pull request:
Updated gemfiles and dependencies to 5.0.2

Tonyynot14 committed:
updated gemfiles with ruby 2.4

micronix commented on Gemfile:
```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.5'
+gem 'rails', '5.0.2'
```
upgrade to latest version

Tonyynot14 replied:
Ramiro I was trying to update to the newest version but could not resolve dependencies without breaking the application because the changed that they have made in between versions. Would you still recommend updating to the newest or can we leave it at 5.0.2, the functionality of the application doesn't require new features

micronix reviewed (commented):
looks good, ready to merge

Tonyynot14 committed:
Upgrade to 5.1.6 rails

Tonyynot14 committed:
Update to 5.2.2.1 rails

Tonyynot14 committed:
Updated to 5.2.2.1 and added info to procedures controller to have facilities loaded in a variable so edit function works

Tonyynot14 commented:
Updated to 5.2.2.1 which is the latest version of rails, also updated small bug on procedure controller not have faculties loaded. I had to do a force push because I used rebase to catch up on changes that others had made. From updating it seems like the browser does not show errors like it used to and says to check the logs. Looking into resolving that

jrmcgarvey reviewed (approved):
for wiki-med, I am going to defer to ramiro, and I will not usually be reviewing pull requests for this project.  However it is good that you are moving up to rails 5.2.2.1.  I'd suggest a couple of additional gems, those being rubocop (for code style checking) and rspec-rails and capybara (for test cases).
